literature.bib

```diff
@@ -13585,6 +14001,19 @@ @article{KKLKKS98
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 1997 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{LPM+:97,
+	author = {B. {Lague} and D. {Proulx} and J. {Mayrand} and E. M. {Merlo} and J. {Hudepohl}},
+	booktitle = ICSM,
+	title = {{Assessing the Benefits of Incorporating Function Clone Detection in a Development Process}},
+	year = 1997,
+	volume = {},
+	number = {},
+	pages = {314--321},
+	keywords = {software cost estimation, software maintenance, software metrics, software quality, telecommunication computing, customer satisfaction, experiment, function clone detection, industrial software development, process change, product assessment, programmers, software clones, software cost, software development process, software maintenance, software quality, source code metrics, telecommunication system},
+	doi = {10.1109/ICSM.1997.624264},
+	ISSN = {1063-6773}
+}
```

Comment:
publisher, address missing

Reply:
Again, I have no idea how to find out which address IEEE is using. :(